app/matplotlib.ipynb

```diff
@@ -159,20 +158,20 @@
   },
   {
    "cell_type": "code",
-   "execution_count": 8,
-   "outputs": [],
    "source": [
     "data = np.random.normal(0, 1, 1000)\n",
     "\n",
-    "# write your code here\n"
+    "plt.hist(data, bins=20, color=\"y\", edgecolor=\"white\")\n",
+    "plt.title(\"x to y\")\n",
+    "plt.xlabel(\"x\")\n",
+    "plt.ylabel(\"y\")\n",
```

Comment:
The histogram's title and axis labels are too generic. Consider using more descriptive labels that reflect the data being plotted.